src/commands/du.rs

```diff
@@ -340,14 +357,19 @@ impl From<DirInfo> for Value {
 impl From<FileInfo> for Value {
     fn from(f: FileInfo) -> Self {
         let mut r: IndexMap<String, Value> = IndexMap::new();
-        r.insert(
-            "size".to_string(),
-            UntaggedValue::bytes(f.size).into_untagged_value(),
-        );
         r.insert(
             "name".to_string(),
             UntaggedValue::string(f.name).into_untagged_value(),
         );
+        r.insert(
+            "size".to_string(),
+            UntaggedValue::bytes(f.size).into_untagged_value(),
+        );
+        let b = match f.blocks {
+            Some(k) => UntaggedValue::int(k).into_untagged_value(),
+            None => UntaggedValue::string("?").into_untagged_value(),
```

Comment:
There's also `UntaggedValue::nothing()` you could use here

Reply:
I figured displaying an unknown was a little more obvious that we couldn't get it for whatever reason.

However.... I just realized that having a string there might cause other things in the pipeline to barf... Would a nothing fix that do you think?

Reply:
Good question. One point for nothing is that you can use the `compact` command to get rid of the rows that contain them.

Reply:
Then I feel that's probably the correct answer. I'll also do a little testing with some other commands to see what barfs and what doesn't.